components/utilities/dialog/index.jsx

```diff
@@ -246,7 +236,8 @@ const Dialog = createReactClass({
 		const { position } = popperData.offsets.popper;
 		const left = `${popperData.offsets.popper.left + propOffsets.horizontal}px`;
 		const top = `${popperData.offsets.popper.top + propOffsets.vertical}px`;
-		return { ...popperData.style, left, top, position };
+		const right = 'inherit';
```

Comment:
@interactivellama just a wip to look at for now, ignore the other changes, I'll drop them if this is acceptable. They're mainly just so i can read and test the code. 

This seems to fix the issue, would like to understand why this can't be added here and whether it'll break other use cases?